Solutions/Marain.UserNotifications.Abstractions/Marain/UserNotifications/UserNotification.cs

```diff
@@ -46,7 +45,7 @@ public class UserNotification
             this.Timestamp = timestamp != default ? timestamp.ToUniversalTime() : throw new ArgumentException(nameof(timestamp));
             this.Properties = properties ?? throw new ArgumentNullException(nameof(properties));
             this.Metadata = metadata ?? throw new ArgumentNullException(nameof(metadata));
-            this.ChannelDeliveryStatuses = channelDeliveryStatuses?.ToImmutableArray() ?? ImmutableArray<UserNotificationStatus>.Empty;
+            this.ChannelStatuses = channelDeliveryStatuses?.ToImmutableArray() ?? ImmutableArray<UserNotificationStatus>.Empty;
```

Comment:
shouldn't the parameter be called `channelStatuses` too?

Reply:
Yes